tea/scenes/new_installation.go

```diff
@@ -0,0 +1,69 @@
+package scenes
+
+import (
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/satisfactorymodding/ficsit-cli/tea/components"
+	"github.com/satisfactorymodding/ficsit-cli/tea/utils"
+)
+
+var _ tea.Model = (*newInstallation)(nil)
+
+type newInstallation struct {
+	root   components.RootModel
+	parent tea.Model
+	input  textinput.Model
+	path   string
+}
+
+func NewInstallation(root components.RootModel, parent tea.Model) tea.Model {
+	model := newInstallation{
+		root:   root,
+		parent: parent,
+		input:  textinput.NewModel(),
+		path:  utils.NonListTitleStyle.Render("/path/to/installation"),
+	}
+
+	model.input.Focus()
+	model.input.Width = root.Size().Width
+
+	return model
+}
+
+func (m newInstallation) Init() tea.Cmd {
+	return nil
+}
+
+func (m newInstallation) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
```

Comment:
As a TUI, it would expected that there is tab-completion and other shell-like features.